addons/field_rations/data/mre.rvmat

```diff
@@ -0,0 +1,32 @@
+ambient[]={1,1,1,1};
```

Comment:
This file should be cleaned up to conform to same config standards.
- Braces on same line
- Spaces around `=`